### PR TITLE
tiltfile is WET: write everything twice

### PR DESCRIPTION
**Pull request by jazzdan**
per discussion here https://app.clubhouse.io/windmill/story/309/servantes-tiltfile-should-repeat-itself

### CODE REVIEW

File: Tiltfile [hunk 1]
```diff
@@ -3,82 +3,101 @@
...
+  image_name = 'gcr.io/windmill-public-containers/servantes/spoonerisms'
 
-def local_file(p):
-  return local("cat %s" % p)
+  img = build_docker_image('Dockerfile.js.base', image_name, 'node /app/index.js')
+  repo = local_git_repo('.')
+  img.add(repo.path('spoonerisms/src'), '/app/')
+  img.add(repo.path('spoonerisms/package.json'), '/app/package.json')
+  img.add(repo.path('spoonerisms/yarn.lock'), '/app/yarn.lock')
+  img.add(repo.path('spoonerisms/'), "/app")
```
Comment (nicks) [inline Tiltfile]: ???

are you sure we want this? this seems to make the previous 3 lines redundant

File: Tiltfile [hunk 2]
```diff
@@ -3,82 +3,101 @@
...
 
-  image_name = 'gcr.io/windmill-public-containers/servantes/%s' % name
+  image_name = 'gcr.io/windmill-public-containers/servantes/vigoda'
 
-  img = build_docker_image('Dockerfile.py.base', image_name, 'python /app/app.py')
+  img = build_docker_image('Dockerfile.go.base', image_name, '/go/bin/vigoda')
+  path = '/go/src/github.com/windmilleng/servantes/vigoda'
   repo = local_git_repo('.')
-  img.add(repo.path('./%s/' % name), "/app")
+  img.add(repo.path('vigoda/'), path)
```
Comment (nicks) [inline Tiltfile]: does it need to end in a slash? what happens if it doesn't?

File: Tiltfile (hunk 1 above)
Comment (jazzdan) [reply Tiltfile]: oh whoops, yeah I don't want that last line.

File: Tiltfile (hunk 2 above)
Comment (jazzdan) [reply Tiltfile]: good question, I'll investigate.

**Pull request merged**